web-bluetooth/scan.js

```diff
@@ -30,6 +30,32 @@ async function onButtonClick() {
 
     navigator.bluetooth.addEventListener('advertisementreceived', event => {
       log('Advertisement received.');
+      log('Device Name: ' + event.device.name);
+      log('  Device ID: ' + event.device.id);
+      log('  RSSI: ' + event.rssi);
+      log('  TX Power: ' + event.txPower);
+      log('  UUIDs: ' + event.uuids);
+
+      const valueDataLogger = (key, valueDataView, whatData) => {
```

Comment:
Please move this at the very bottom in `/* utils */`.
See https://googlechrome.github.io/samples/web-bluetooth/device-information-characteristics.html for instance.

Reply:
Done